### DISCUSSION TITLE
Contour Fill

### DISCUSSION THREAD

**Discussion by t-tttt** [inkstitch/inkstitch#1494] [General]
Thank you for putting all the time that you all have into creating inkstitch!

We have done quite a bit of embroidery with inkstitch. Just the nature of embroidery itself can make designs/images look flat/2D. We have tried to off set this by using satin stitches for everything. I am wondering if there is a way to be able to contour the fill pattern to give a design more depth. I think that would give inkstitch a whole other level professionalism.

Thanks again for all the work you developers have done!

**Comment by tabbzi** [comment]
I believe the devs working on this as a Feature Requests project from issue #1359. It's not on main, but there's a branch feature_guided_fill with the in-progress feature.

Seconded, I'm really looking forward to this feature! Loving InkStitch, it's been a godsend for digitizing.
